src/common/options.cc

```diff
@@ -3876,7 +3877,7 @@ std::vector<Option> get_global_options() {
 
     Option("osd_recovery_op_priority", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
As any other priorities, it is relative to WHAT ?  priority over client's operations ? other OSD's ? other pools ?

Reply:
The op priorities are used by the message layer in some way.  The recovery messages are the ones that control the recovery process.  They are things like MOSDPGPush (object data), MOSDPGPushReply, MOSDPGBackfill (control messages).  The osd_client_op_priority (default 63) is larger so that internal messages get priority over recovery messages (default 3)